core/src/main/scala/org/http4s/rho/RhoDslPathExtractors.scala

```diff
@@ -34,3 +42,13 @@ trait RhoDslPathExtractors[F[_]] {
     TypedPath(PathCapture[F](id, Some(description), parser, stringTag))
 
 }
+
+object RhoDslPathExtractors {
+
+  private val stringTag = implicitly[TypeTag[String]]
+
+  class PathCaptureStringContext[F[_]](val sc: StringContext) extends AnyVal {
+    def pv(args: Any*): TypedPath[F, String :: HNil] =
```

Comment:
So, I'm not sure about allowing args to the interpolater.

Are we ok with path names like this? 

```
val foo = "Some string from somewhere"

GET / "helloworldstring" / pv"world-$foo"

``` 

If this is desirable I'm OK with it, by I feel like we should restrict it to just other strings instead of `Any`?

Reply:
For some reason I was convinced that to use that syntax I must define a method with `args: Any*`. I just learned that this isn't true. 

I would prefer to completely disable the interpolation.